tests/src/test/scala/ch/epfl/directembedding/test/ErrorsSpec.scala

```diff
@@ -9,11 +9,20 @@ class ErrorsSpec extends FlatSpec with ShouldMatchers with ExampleTester {
   "dsl" should "give a useful error when a reifyAs annotation is missing" in {
     typedWithMsg(
       """
-        runTest(
-          dsl {
-            ObjectExample.missingAnnotation
-        }) should be(???)
-      """, "method missingAnnotation is not supported in example.dsl")
+        dsl {
+          ObjectExample.missingAnnotation
+        }
+      """, "method missingAnnotation on object ObjectExample is not supported in example.dsl")
```

Comment:
The position in the error message is now also on the invoked symbol instead of the outer `dsl` block, a big improvement.